### PR TITLE
Improve the configuration guide

### PR DESCRIPTION
**Pull request by adenosie**
- Fix some typoes
- Add Korean translation(`CONFIGURATION.ko.md`)
- Make default value easier to read

### CODE REVIEW

File: docs/CONFIGURATION.ko.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# 설정
+
+[🇺🇸](CONFIGURATION.md), [🇰🇷](CONFIGURATION.ko.md)
+
+`/etc/kime/config.yaml`에 기본값으로 설정된 파일이 있습니다. [default_config.yaml](default_config.yaml)에서 기본 설정 파일을 온라인으로 볼 수도 있습니다. `/etc/kime/config.yaml`에서 전역 설정을 수정하거나 `~/.config/kime/config.yaml`에 새 설정 파일을 만들어 보세요.
+
+[`$XDG_CONFIG_DIR`이나 `$XDG_CONFIG_HOME`][xdg] 환경 변수를 이용해 설정 파일의 위치를 바꿀 수도 있습니다. kime는 `$XDG_CONFIG_DIR/kime/config.yaml`과 `$XDG_CONFIG_HOME/kime/config.yaml`에 있는 설정 파일도 읽으려고 시도할 것입니다.
+
+[xdg]: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html#introduction
```
Comment (Riey) [inline docs/CONFIGURATION.ko.md:9]: Where is this link used? I can't see in rendered document

File: docs/CONFIGURATION.ko.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+## layout
+
+키보드 자판을 설정합니다. `dubeolsik`(두벌식), `sebeolsik-390`(세벌식 390), `sebeolsik-391`(세벌식 391)이 기본으로 내장되어 있습니다. `$XDG_CONFIG_HOME/kime/layouts/`에 위 목록에 없는 키보드 자판을 YAML 파일로 직접 만들 수도 있습니다. [dubeolsik.yaml]을 참고해 보세요.
+
+[dubeolsik.yaml]: engine/core/data/dubeolsik.yaml
```
Comment (Riey) [inline docs/CONFIGURATION.ko.md:15]: Link broken in eng version either

```suggestion
[dubeolsik.yaml]: ../engine/core/data/dubeolsik.yaml
```

File: docs/CONFIGURATION.ko.md (hunk 1 above)
Comment (Riey) [inline docs/CONFIGURATION.ko.md:9]: I can't see in rendered document

Comment (adenosie) [reply docs/CONFIGURATION.ko.md:9]: The link works well for me..

Comment (Riey) [reply docs/CONFIGURATION.ko.md:9]: Oh sorry I didn't know that grammar properly.

**Review by Riey — APPROVED** [review]

**Pull request merged**